consensus/bor/heimdallapp/client.go

```diff
@@ -21,7 +18,8 @@ type HeimdallAppClient struct {
 
 func NewHeimdallAppClient() *HeimdallAppClient {
 	return &HeimdallAppClient{
-		hApp: service.GetHeimdallApp(),
+		// TODO HV2: Implement according to the new setup
+		// hApp: service.GetHeimdallApp(),
```

Comment:
Not needed now? Btw the function itself seems to be unused with v2, while is being used in `develop` when creating the consensus engine. What's the diff here?

Reply:
This is used when running heimdall from bor, changed the comment and added panic.